content/english/anti-vegan/pests.md

```diff
@@ -0,0 +1,150 @@
+
+[Department of Conservation NZ](http://web.archive.org/save/https://www.doc.govt.nz/nature/pests-and-threats/animal-pests/possums/) lists possums as pests, as, “Leaves are the main part of their diet, but possums are opportunistic omnivores. They eat buds, flowers, fruit/berries and nectar, which means they compete with native birds and reptiles for food sources. The growth and life-cycle of a tree or plant is significantly affected when all parts of it are eaten. Possums also have ‘favourites’ such as rātā or kamahi trees, leading to an even greater impact on these species.  …Possums were [also]filmed eating the eggs and chicks of kōkako …  They eat invertebrates, including weta, and are significant predators of New Zealand land snails such as Powelliphanta. They often occupy holes in tree trunks for their nests which would otherwise be used by nesting birds such as kākāriki and saddlebacks. …Researchers using nest-cameras have witnessed … kea nests invaded by stoats and possums in South Westland. ”
+
+Here it would seem possums can be defined as a ‘pest species’ that directly impacts the ecosystem, without any reference to human specific interests.
+
+### To recap
+
+Vegans may still object to the term 'pest'. A species that is out-of-place in the eco-system is, in most cases, due to past meddling by humans and not a fault of the species. 
+
+Vegans can classify a species as problematic by reference to Leopold's non-speciesist definition. However, once identified, any action taken must *as far as is possible and practicable* be neither exploitive nor unnecessarily cruel.
+
+### So where to from here? 
+
+While the ideals of veganism are always placed first, individual vegans will differ on what, if any, actions ought to be taken. This can range from 'do nothing' (*all species have a right to life*) to 'controlling possums' (applying the golden rule of *acting to bring about the least harm*)
+
+This is discussed in more depth in the appendix.
+
+### Final thoughts
+
+Identifying a species as problematic does not commit a vegan to act in any specific manner. However, that vegans can offer a non-speciesist definition of 'pest' and can argue for a specific approach in dealing with them demonstrates that vegans can address complex environmental concerns.
+
+While there may be disagreement amongst vegans regarding the moral course of action there is agreement that any actions must *as far as is possible and practicable* not be exploitive or unnecessarily cruel.
+
+This is no different from individuals facing any difficult complex moral questions. Conflicts of opinion arise over euthanasia, abortion, freedom of speech, and many others.
+
+## Appendix.
+
+### Warning: Going Philosophical
+
+Sorry, but the next section gets a little academic. The author believes it necessary but it's the sort of stuff that causes some people to gnaw their own limbs off. If you are interested in the philosophical underpinning great... read on...
```

Comment:
Let's not apologise! Just warn.
How about
Title: 
> Fitting all this into ethical philosophy
Disclaimer: 
> This section discusses ethical theory in some depth. Dive in if you're interested, but the arguments above should be enough to serve you in your day-to-day discussions.

Reply:
Ahhh. can I keep douglas adams